modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -3,10 +3,20 @@
 // * monitoring/configuring-the-monitoring-stack.adoc
 
 :_content-type: PROCEDURE
-[id="modifying-retention-time-for-prometheus-metrics-data_{context}"]
-= Modifying the retention time for Prometheus metrics data
+[id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
+= Modifying the retention time and size for Prometheus metrics data
 
-By default, the {product-title} monitoring stack configures the retention time for Prometheus data to be 15 days. You can modify the retention time to change how soon the data is deleted.
+By default, Prometheus automatically retains metrics data for 15 days.
+You can modify the retention time to change how soon data is deleted.
+You can also configure the maximum amount of disk space used by the retained metrics data.
+If this size limit is reached, the oldest data is deleted first until the disk space used by the data is again below the maximum.
+
+Note the following behaviors of these configuration settings:
+
+* If you do not explicitly define values for either the retention time or retention size fields, retention time defaults to 15 days, and retention size is not set.
+* If you define values for both retention time and retention size fields, both values apply.
```

Comment:
If you define values for both retention time and retention size fields, both values apply. If any of them is true at some point in time (either blocks are too old or the size of all blocks exceeds the size limit) then data is purged.